src/composables/pollApi.js

```diff
@@ -95,5 +117,8 @@ export const pollApi = () => {
         router.push('/pollList')
     }
 
-    return {  polls,user ,userToken, countVote, isState, showAddPoll,  addNewPoll, newPoll, addOptions, option, addError, showPoll, poll, viewPolls }
+    return {
+        polls, user, userToken, countVote, isState, showAddPoll, addNewPoll, newPoll,
+        deleteNewopt, updateNewopt, addOptions, option, addError, showPoll, poll, viewPolls
```

Comment:
poll composable should not return user, userToken. Composable should only return only for a specific module.
If you need user related stuff, create a authComposable or userComposable and use it from there @PiyushInnotech